ts-microservices/test/pulumi/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as aws from "@pulumi/aws";
```

Comment:
oh this is nifty, there's stack validation. If we have diff klotho configs that we're testing we'll need accompanying policy packs right?

Reply:
yup